templates/docs/examples/patterns/code-snippet/code-snippet-numbered.html

```diff
@@ -0,0 +1,23 @@
+{% extends "_layouts/examples.html" %}
+{% block title %}Code snippet / Numbered{% endblock %}
+
+{% block standalone_css %}patterns_code-snippet{% endblock %}
+
+{% block content %}
+<div class="p-code-snippet">
+  <div class="p-code-snippet__header">
+    <h5 class="p-code-snippet__title">Check you have snap installed</h5>
```

Comment:
The header doesn't seems to be correct for the code content. Update it (but I don't know what the code is doing) ;) or remove the headier.